packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+import ActionHeaderContainer from './_ActionHeaderContainer';
+
+const propTypes = {
+  /**
+   * Text to be displayed as the title in the header bar.
+   */
+  title: PropTypes.string,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. Default `level=1`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
I know we have used prop name as level for section header, but is it not better to name it as headingLevel?

Reply:
The `level` prop so far has only been used for components that invoke a heading h1-h6 element, so it is currently consistent with all other components thus far as is. Only risk is if level is introduced elsewhere for another concept in the future - then we would want to adjust all others as well .